account_mass_reconcile/models/base_advanced_reconciliation.py

```diff
@@ -217,9 +218,9 @@ def _skip_line(self, move_line):
         """
         return False
 
-    @api.multi
     def _rec_auto_lines_advanced(self, credit_lines, debit_lines):
         """ Advanced reconciliation main loop """
+        # pylint: disable=invalid-commit
```

Comment:
why? what does it mean?

Reply:
To avoid Travis from failing due to commits done inside this function